frontend/components/panels/HighestDebtBidCheckPanel.vue

```diff
@@ -0,0 +1,153 @@
+                    >Bid <FormatCurrency :value="bidAmount" currency="DAI" /> for
+                    <FormatCurrency :value="auction.receiveAmountMKR" currency="MKR"
+                /></span>
+                <span v-else>Unknown</span>
+            </BaseButton>
+        </div>
+    </BasePanel>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { DebtAuction } from 'auctions-core/src/types';
+import BigNumber from 'bignumber.js';
+import BasePanel from '~/components/common/other/BasePanel.vue';
+import TextBlock from '~/components/common/other/TextBlock.vue';
+import BaseButton from '~/components/common/inputs/BaseButton.vue';
+import FormatCurrency from '~/components/common/formatters/FormatCurrency.vue';
+
+export default Vue.extend({
+    name: 'HighestDebtBidCheckPanel',
+    components: {
+        FormatCurrency,
+        BaseButton,
+        TextBlock,
+        BasePanel,
+    },
+    props: {
+        auction: {
+            type: Object as Vue.PropType<DebtAuction>,
+            default: null,
+            required: true,
+        },
+        walletAddress: {
+            type: String,
+            default: null,
+        },
+        bidAmount: {
+            type: Object as Vue.PropType<BigNumber>,
+            default: undefined,
+        },
+        isBidding: {
+            type: Boolean,
+            default: false,
+        },
+        disabled: {
+            type: Boolean,
+            default: false,
+        },
+        isLoading: {
+            type: Boolean,
+            default: false,
+        },
+        error: {
+            type: String,
+            default: null,
+        },
+        isExplanationsShown: {
+            type: Boolean,
+            default: true,
+        },
+    },
+    computed: {
+        wasThereAnyBids(): boolean {
+            if (!this.auction.bidAmountDai) {
+                return false;
+            }
+            return !this.auction.bidAmountDai.isZero();
+        },
+        isUserLatestBidder(): boolean {
+            return (
+                this.wasThereAnyBids &&
+                this.auction.receiverAddress &&
+                this.walletAddress &&
+                this.auction.receiverAddress.toLowerCase() === this.walletAddress.toLowerCase()
+            );
+        },
+        currentStateAndTitle(): PanelProps {
+            if (this.auction.state === 'collected') {
+                return {
+                    name: 'inactive',
+                    title: `The auction has finished and been collected`,
+                };
+            }
+            if (!this.wasThereAnyBids) {
+                return {
+                    name: 'notice',
+                    title: 'No bids yet',
+                };
+            }
+            if (this.auction.state !== 'ready-for-collection') {
+                if (this.isUserLatestBidder) {
+                    return {
+                        name: 'correct',
+                        title: 'You are the highest bidder',
```

Comment:
To not confuse users with `lowest`, it can be

```suggestion
                        title: 'You are the latest bidder',
```
or `winning bidder`

@LukSteib?

(same applies to at least 2 other places below with `highest` in the text)

Reply:
I prefer latest!

Reply:
Yep agree with `latest` -> as indicated in the mock #397
Thx for catching